packages/react-form-state/src/test/FormState.test.tsx

```diff
@@ -652,8 +682,75 @@ describe('<FormState />', () => {
       expect(descriptionError.message).toEqual(submitErrors[1].message);
     });
   });
+
+  describe('performance', () => {
+    it('does not re-render form when state after onchange is identical', () => {
+      const product = faker.commerce.productName();
+      const description = faker.lorem.words();
+
+      const renderPropSpy = jest.fn(() => null);
+
+      mount(
+        <FormState initialValues={{product, description}}>
```

Comment:
You don't need description, just distracts from the purpose of the test

Reply:
I do need it in this object since we are testing whether the description field rerenders. I can make this value just an empty string though.